### PR TITLE
 change of position from the entered color

### PR DESCRIPTION
**Pull request by paxarpp**

### CODE REVIEW

File: src/rgbToHsv.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/**
```
Comment (ubcent) [inline src/rgbToHsv.jsx]: It should has .js extension

File: src/index.jsx [hunk 2]
```diff
@@ -20,11 +21,13 @@ class CircularColor extends PureComponent {
...
+    const { h,s,v } = rgbToHsv(color);
+
     this.state = {
       touched: false,
-      cx: Math.floor((size / 2) + (innerRadius + handlerSize) * Math.cos(Math.PI)),
-      cy: Math.floor((size / 2) - (innerRadius + handlerSize) * Math.sin(Math.PI)),
-      color: hsvToRgb(Math.abs(Math.PI) * 180 / Math.PI)
+      cx: Math.floor((size / 2) + (innerRadius + handlerSize) * Math.cos((Math.PI*h)/180)),
+      cy: Math.floor((size / 2) - (innerRadius + handlerSize) * Math.sin((Math.PI*h)/180)),
+      color: color,
```
Comment (ubcent) [inline src/index.jsx]: I guess it should be shorthand assignment

File: src/index.jsx [hunk 3]
```diff
@@ -20,11 +21,13 @@ class CircularColor extends PureComponent {
       outerRadius
     };
 
+    const { h,s,v } = rgbToHsv(color);
```
Comment (ubcent) [inline src/index.jsx]:
```javscript
const { h, s, v } = rgbToHsv(color);
```

File: src/index.jsx [hunk 4]
```diff
@@ -20,11 +21,13 @@ class CircularColor extends PureComponent {
...
+
     this.state = {
       touched: false,
-      cx: Math.floor((size / 2) + (innerRadius + handlerSize) * Math.cos(Math.PI)),
-      cy: Math.floor((size / 2) - (innerRadius + handlerSize) * Math.sin(Math.PI)),
-      color: hsvToRgb(Math.abs(Math.PI) * 180 / Math.PI)
+      cx: Math.floor((size / 2) + (innerRadius + handlerSize) * Math.cos((Math.PI*h)/180)),
+      cy: Math.floor((size / 2) - (innerRadius + handlerSize) * Math.sin((Math.PI*h)/180)),
```
Comment (ubcent) [inline src/index.jsx]:
```javascript
cx: Math.floor((size / 2) + (innerRadius + handlerSize) * Math.cos((Math.PI * h) / 180)),
cy: Math.floor((size / 2) - (innerRadius + handlerSize) * Math.sin((Math.PI * h) / 180)),
```

File: src/index.jsx [hunk 5]
```diff
@@ -178,13 +181,15 @@ CircularColor.propTypes = {
...
+  centerRect: PropTypes.bool,
+  color: PropTypes.string
 };
 
 CircularColor.defaultProps = {
   size: 200,
   numberOfSectors: 360,
-  centerRect: false
+  centerRect: false,
+  color: '#f2ff00'
```
Comment (ubcent) [inline src/index.jsx]: We need trailing comma here according to style guide

**Comment by ubcent** [comment]
BTW, @paxarpp what do u think about changing our custom hsvToRGB and so on to 3rd party module ["color"](https://www.npmjs.com/package/color)

File: src/rgbToHsv.jsx (hunk 1 above)
Comment (paxarpp) [reply src/rgbToHsv.jsx]: with the extension .js webpack did not compile the project (did not find this file)

Comment (ubcent) [reply src/rgbToHsv.jsx]: Actually, it sounds strange because hsvToRGB has the .js extension

**Comment by paxarpp** [comment]
why not

File: src/rgbToHsv.jsx (hunk 1 above)
Comment (paxarpp) [reply src/rgbToHsv.jsx]: I know that it's strange, maybe a Windows?

Comment (paxarpp) [reply src/rgbToHsv.jsx]: client?8505:167 ./src/rgbToHsv.jsx
Module build failed: Error: ENOENT: no such file or directory, open 'D:\KODING\issue\react-circular-color\src\rgbToHsv.jsx'
 @ ./src/index.jsx 25:17-38

Comment (paxarpp) [reply src/rgbToHsv.jsx]: hmm, now all ok

**Pull request merged**